### PR TITLE
Fix build process for KAVA, DGLD, ARBETH

### PR DESCRIPTION
**Pull request by pavel-bc**
Replacement for https://github.com/blockchain/coin-definitions/pull/116

### CODE REVIEW

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [ARG.CHZ, ERC20]: displayed as: ARG.CHZ
-  ⚠️  [BCH, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [DGLD.ETH, ERC20]: displayed as: DGLDn
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.003 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [POR.CHZ, ERC20]: displayed as: POR.CHZ
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC
-  ⚠️  [SOL, COIN]: minWithdrawal 0.1 -> $0.000 not in the $0.01-$10 USD range

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [ARG.CHZ, ERC20]: displayed as: ARG.CHZ
-  ⚠️  [BCH, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [DGLD.ETH, ERC20]: displayed as: DGLDn
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.003 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [POR.CHZ, ERC20]: displayed as: POR.CHZ
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC
-  ⚠️  [SOL, COIN]: minWithdrawal 0.1 -> $0.000 not in the $0.01-$10 USD range

File: extensions/blockchains/ethereum/assets/0xA9299C296d7830A99414d1E5546F5171fA01E9c8/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "DGLD",
```
Comment (ndcotta-bc) [inline extensions/blockchains/ethereum/assets/0xA9299C296d7830A99414d1E5546F5171fA01E9c8/info.json:2]:
```suggestion
  "name": "Digital Gold",
```

or is DGLD preferable?

**Review by ndcotta-bc — APPROVED** [review]

File: extensions/blockchains/ethereum/assets/0xA9299C296d7830A99414d1E5546F5171fA01E9c8/info.json (hunk 1 above)
Comment (pavel-bc) [reply extensions/blockchains/ethereum/assets/0xA9299C296d7830A99414d1E5546F5171fA01E9c8/info.json]: Good catch, updated

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [ARG.CHZ, ERC20]: displayed as: ARG.CHZ
-  ⚠️  [BCH, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [DGLD.ETH, ERC20]: displayed as: DGLDn
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.003 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [POR.CHZ, ERC20]: displayed as: POR.CHZ
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC
-  ⚠️  [SOL, COIN]: minWithdrawal 0.1 -> $0.000 not in the $0.01-$10 USD range

**Review by ndcotta-bc — APPROVED** [review]

**Pull request merged**